documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -100,13 +275,13 @@ Note: the examples are written in RDF/Turtle syntax, and only "Hello" and "Hallo
 
 |===
 
-The following types defined by the XSD and RDF specifications are explicitly omitted for serialization:
+The following types defined by the XSD and RDF specifications are explicitly omitted for serialization. This is ensured because they are not part of enumerations xref:IDTA-01001_Metamodel_DataTypes.adoc#[DataTypeDefXsd] and xref:IDTA-01001_Metamodel_DataTypes.adoc#[DataTypeDefRdf]:
 
 xs:language, xs:normalizedString, xs:token, xs:NMTOKEN, xs:Name, xs:NCName, xs:QName, xs:ENTITY, xs:ID, xs:IDREF, xs:NOTATION, xs:IDREFS, xs:ENTITIES, xs:NMTOKENS, rdf:HTML and rdf:XMLLiteral.
 
 
 ====
-Note 1: due to the limits in the representation of numbers in JSON, the maximum integer number that can be used without losing precision is 2⁵³-1 (defined as Number.MAX_SAFE_INTEGER). Even if the used data type would allow higher or lower values, they cannot be used if they cannot be represented in JSON. Affected data types are unbounded numeric types xs:decimal, xs:integer, xs:positiveInteger, xs:nonNegativeInteger, xs:negativeInteger, xs:nonPositiveInteger and the bounded type xs:unsignedLong. Other numeric types are not affected. footnote:[cf. https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/datatypes.html (with adjustments for +/-INF, NaN, and language-typed literal support)]
+Note 1: due to the limits in the representation of numbers in JSON, the maximum integer number that can be used without losing precision is 2⁵³-1 (defined as Number.MAX_SAFE_INTEGER). Even if the used data type would allow higher or lower values, they cannot be used if they cannot be represented in JSON. Affected data types are unbounded numeric types xs:decimal, xs:integer, xs:positiveInteger, xs:nonNegativeInteger, xs:negativeInteger, xs:nonPositiveInteger and the bounded type xs:unsignedLong. Other numeric types are not affected. footnote:[cf. https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html (with adjustments for +/-INF, NaN, and language-typed literal support)]
```

Comment:
I am not an expert, but where is `Number.MAX_SAFE_NUMBER` defined? This seems to be implementation-specific, i.e. depend on the runtime/framework/programming language used. JSON itself does not define any boundaries for numbers. Please provide are clearer explanation and/or a source.

Reply:
Link from ESMF: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER added + link corrected

Reply:
The provided link does not back what is claimed here. First, it has no connection at all to JSON; the word JSON is not even mentioned in the linked document. Second, it explicitly says in the first sentence, that this only applies to JavaScript.
Please add a source (or at least detailed explanation) for the following statement 
> due to the limits in the representation of numbers in JSON, the maximum integer number that can be used without losing precision is 2⁵³-1 

I am not against having this statement per-se, but as I never have heard of this and currently don't understand why this would be the case I think this requires proof.